apps/files_reminders/lib/Listener/LoadAdditionalScriptsListener.php

```diff
@@ -15,11 +15,13 @@
 use OCP\EventDispatcher\Event;
 use OCP\EventDispatcher\IEventListener;
 use OCP\Util;
+use Psr\Log\LoggerInterface;
 
 /** @template-implements IEventListener<LoadAdditionalScriptsEvent> */
 class LoadAdditionalScriptsListener implements IEventListener {
 	public function __construct(
 		private IAppManager $appManager,
+		private readonly LoggerInterface $logger,
```

Comment:
```suggestion
		private LoggerInterface $logger,
```
If you want to backport this because 28 still supports PHP 8.0 :)

Reply:
Done!